app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
+
+  def recent_posts(limit = 3)
+    posts.order(created_at: :desc).limit(limit)
+  end
```

Comment:
- [ ] Great catch! The current implementation of the recent_posts method is almost there, but to ensure consistency, it's crucial that the method always returns the 3 most recent posts. Since the default value of the limit parameter can be accidentally overridden when calling the method in your implementation, I recommend keeping the limit value internal to the method. By doing so, you guarantee that the method consistently returns the desired 3 most recent posts, regardless of any arguments passed.
- Here's an example 
```rb
def recent_post
  posts.order('created_at Desc').limit(3)
end
```